### DISCUSSION TITLE
Huawei HG8244H ONR

### DISCUSSION THREAD

**Discussion by MrSebBird** [Anime4000/RTL960x#163] [Q&A]
Firstly to all the collaborators, thank you for your efforts.

I am wondering if i can achieve connectivity directly to a Mikrotik 5009 with a ODI DFP-34X-2C2

Currently, the supplied NTU is a Huawei HG8244H ONR, i have access to all settings however I am wondering since this is an ONR and not an ONT if I am still able to achieve connectivity back to my ISP?

Has anyone had success with the Huawei HG8244H ONR?

**Comment by Anime4000** [comment]
you just clone all OMCI and transfer to ODI Stick, OLT will see as normal ONR and ODI Firmware will bridge it as is, if using VLAN, just VLAN at router side and PPPoE (if any)

**Comment by MrSebBird** [comment]
Thank you for the quick response.

Unfortunately, i am still stuck at 03

I have set the following.

```
PON_VENDOR_ID=HWTC

HW_CWMP_MANUFACTURER=HWTC

HW_CWMP_PRODUCTCLASS=HGU

HW_HWVER=714.E

OMCI_VEIP_SLOT_ID=255

OMCI_OLT_MODE=3

OMCI_FAKE_OK=1

OMCI_SW_VER1=V1.7.1 (this will not set to the one listed in the ONR even with OMCI_OLT_MODE=3 set)

OMCI_SW_VER2=V1.0-220817 (this will not set to the one listed in the ONR even with OMCI_OLT_MODE=3 set)

GPON_ONU_MODEL=HGU

GPON_SN=HWTCXXXXXXXX (Redacted)

GPON_PLOAM_PASSWD=(10 CHAR FROM FTTH ID Of ONR Auth section of HG8244H)

ELAN_MAC_ADDR=default

# flash get VS_AUTH_KEY

[ERR] Warning, unknown dir name: VS_AUTH_KEY

GET fail.
# flash get MAC_KEY

MAC_KEY=default  (Do i need to set this to the mac of the ONR?)

OUI=first 6 digits and letters of MAC

HW_SERIAL_NO=HWTCXXXXXXXX (Redacted)
```

I have tried the firmware

- M110_sfp_ODI_220304.tar
- M110_sfp_ODI_220923.tar
- M110_sfp_ODI_220817.tar

Happy to post screen shots of my ONR if that would help

Thanks again
